lib/components/ScrollView.vue

```diff
@@ -0,0 +1,41 @@
+<template>
```

Comment:
On web, given the API, this component should be used to restrict which way the scroll works. By defaultm scrolling works both ways on web, when set to "horizontal", we should rather restrict vertical scrolling and vice versa.